### PR TITLE
Docs use diagram in app

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] Docs: How to use diagram in ClearCalcs

### CODE REVIEW

File: docs/using-in-clearcalcs.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Using in ClearCalcs
+
+## Sign Up
+
+ClearCalcs is a platform for structural engineers to use calculators to design safe structures such as buildings and civil infrastructure. The calculators are creatd
```
Comment (JayserBeam) [inline docs/using-in-clearcalcs.md:5]: created*

Comment (JayserBeam) [reply docs/using-in-clearcalcs.md:5]: Looks like the train of thought left the station early

**Review by JayserBeam — CHANGES_REQUESTED** [review]
Everything else looks fine to me. Ping me and i'll re approve once the paragraph is completed.

File: docs/using-in-clearcalcs.md (hunk 1 above)
Comment (michaellisitsa) [reply docs/using-in-clearcalcs.md]: Nice pickup. updated in e9af6bd92c52edef032995672350641df0948630

**Review by JayserBeam — APPROVED** [review]
Added the remainder of the intro paragraph. I am able to follow along with the instructions. LGTM

**Pull request merged**